frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/QuestionCards/CamperQuestionsCard.tsx

```diff
@@ -31,7 +31,7 @@ const CamperQuestionsCard = ({
         <Wrap width="100%" px="20px" justify="space-between">
           {campSpecificFormQuestions.map((question) => (
             <WrapItem
-              key={`additional_info_question_${question}`}
+              key={`additional_info_question_${question.id}`}
```

Comment:
With template strings, ${object} won't display the entire object data so we end up having duplicate keys that all say "additional_info_question_[object]" or something of that sort, so we need the id.

Reply:
yeah good catch. I also made the same change on my feature branch but I'll just update it to match this change